package.json

```diff
@@ -1,48 +1,45 @@
 {
+  "homepage": "https://github.com/strimzi/strimzi-ui#readme",
+  "license": "Apache-2.0",
   "name": "strimzi-ui",
   "private": true,
-  "version": "0.0.1",
-  "description": "Strimzi UI provides a way for managing Strimzi and Kafka clusters (+ other components) deployed by it using a graphical user interface.",
-  "scripts": {
-    "clean": "rm -rf dist/ && rm -rf generated/",
-    "start": "webpack-dev-server --config build/webpack.dev.js",
-    "build": "webpack --config build/webpack.prod.js",
-    "server": "",
-    "test:unit": "jest --collectCoverage --config test_common/jest.config.js",
-    "test:e2e": "npm-run-all clean build server cy:closed",
-    "test:e2e:open": "npm-run-all clean build server cy:open",
-    "pretest": "npm-run-all --parallel licence:check && prettier --config ./linting/prettier.config.js --check .",
-    "storybook": "start-storybook --port 6006",
-    "storybook:build": "build-storybook -o .out",
-    "licence:check": "license-check-and-add --f ./linting/license-check-and-add.config.json check",
-    "licence:add": "license-check-and-add --f ./linting/license-check-and-add.config.json add",
-    "lint": "npm-run-all 'lint:allfiles -- .' 'lint:js -- .' 'lint:styling -- \"**/*.(s)css\"' 'lint:format -- .'",
-    "lint:allfiles": "npm-run-all 'licence:add -- {1}' --",
-    "lint:js": "eslint --config ./linting/eslint.config.js --fix --max-warnings 0 --ignore-path ./linting/.eslintignore",
-    "lint:styling": "stylelint --config ./linting/stylelint.config.js --fix --max-warnings 0",
-    "lint:format": "prettier --config ./linting/prettier.config.js --write",
-    "cy:open": "cypress open --config-file e2e/cypress.json",
-    "cy:closed": "cypress run --config-file e2e/cypress.json"
-  },
-  "sideEffects": false,
   "repository": {
     "type": "git",
     "url": "git+https://github.com/strimzi/strimzi-ui.git"
   },
-  "license": "Apache-2.0",
+  "description": "Strimzi UI provides a way for managing Strimzi and Kafka clusters (+ other components) deployed by it using a graphical user interface.",
+  "sideEffects": false,
+  "version": "0.0.1",
   "bugs": {},
-  "homepage": "https://github.com/strimzi/strimzi-ui#readme",
   "dependencies": {
     "@babel/core": "^7.10.5",
     "@babel/preset-env": "^7.10.4",
+    "@typescript-eslint/eslint-plugin": "^4.4.1",
     "babel-loader": "^8.1.0",
     "compression-webpack-plugin": "^4.0.0",
+    "eslint": "^7.6.0",
```

Comment:
as a `dependencies`?

Reply:
eslint is the runtime, which rule checks. Plugins modify. Worth confirming eslint itself doesnt bundle plugins. If it does, then it should be a dep by our definition